addons/mail/static/src/models/message_context_menu.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one, many, attr } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'MessageContextMenu',
+    recordMethods: {
+
+        
+        /**
+         * Returns whether the given html element is inside this message context menu.
+         *
+         * @param {Element} element
+         * @returns {boolean}
+         */
+        containsElement(element) {
+            return Boolean(this.component && this.component.root.el && this.component.root.el.firstChild.contains(element));
+        },
+    },
+    fields: {
+        messageContextMenuDialog: one('Dialog', {
+            identifying: true,
+            inverse: 'messageContextMenu',
+        }),
+        messageViewOwner: one('MessageView', {
+            related: 'messageContextMenuDialog.messageViewOwnerAsContextMenu'
+        }),
```

Comment:
Makes more sense to define field `message` and just rely on it. Don't use `MessageView` in the logic of this model. Or at the very least, minimise it's use e.g. just for computing `message` field.